code/game/objects/items/documents.dm

```diff
@@ -115,3 +221,218 @@
+		to_chat(user, "<span class='info'>\The [src] consumes \the [I]!</span>")
+		paper_charges+=charges_per_paper
+		if(paper_charges>max_paper_charges)
+			paper_charges = max_paper_charges
+		qdel(I)
+
+/obj/item/inspector/clown/bananium/Initialize()
+	. = ..()
+	playsound(src, 'sound/effects/angryboat.ogg', 150, FALSE)
+
+/obj/item/inspector/clown/bananium/create_slip()
+	if(print_time == 0.1 SECONDS)
+		var/obj/item/paper/fake_report/water/slip = new(get_turf(src))
+		slip.generate_report(get_area(src))
+	else
+		..()
+
+/obj/item/inspector/clown/bananium/print_report(mob/user)
+	if(print_time != 0.1 SECONDS)
+		..()
+	else if(paper_charges == 0)
+		if(cell.use(power_to_speak))
+			say("ERROR! OUT OF PAPER! MAXIMUM PRINTING SPEED UNAVAIBLE! SWITCH TO A SLOWER SPEED TO OR PROVIDE PAPER!")
+		else
+			to_chat(user, "<span class='info'>\The [src] doesn't seem to be on... Perhaps it ran out of power?")
+	else
+		paper_charges--
+		..()
+
+/obj/item/inspector/clown/bananium/cycle_print_time(mob/user)
+	var/message = "You turn the screw-like dial, setting the device's scanning speed to "
+	if(print_time == 0.1 SECONDS)
+		power_per_print = INSPECTOR_POWER_USAGE_NORMAL
+		print_time = 5 SECONDS
+		message += "SLOW."
+	else if(print_time == 5 SECONDS)
+		print_time = 1 SECONDS
+		message += "LIGHTNING FAST."
+	else
+		print_time = 0.1 SECONDS
+		power_per_print = INSPECTOR_POWER_USAGE_HONK
+		message += "HONK!"
+	balloon_alert(user, message)
+
+/**
+ * Reports printed by fake N-spect scanner
+ *
+ * Not valid for the bounty.
+ */
+/obj/item/paper/fake_report
+	name = "encrypted station inspection"
+	desc = "Contains no information about the station's current status."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "slip"
+	///What area the inspector scanned when the report was made. Used to generate the examine text of the report
+	var/area/scanned_area
+	show_written_words = FALSE
+
+/obj/item/paper/fake_report/proc/generate_report(area/scan_area)
+	scanned_area = scan_area
+	icon_state = "slipfull"
+
+	var/list/characters = list()
+	characters += GLOB.alphabet
+	characters += GLOB.alphabet_upper
+	characters += GLOB.numerals
+
+	var/length = rand(23, 123)
+	var/i
+	for(i = 0; i<length; i++)
+		if(prob(90))
+			info += pick_list_replacements(CLOWN_NONSENSE_FILE, "honk")
+		else if(prob(1))
+			info += pick_list_replacements(CLOWN_NONSENSE_FILE, "rare")
+		else
+			info += pick_list_replacements(CLOWN_NONSENSE_FILE, "non-honk-clown-words")
+
+/obj/item/paper/fake_report/examine(mob/user)
+	. = ..()
+	if(scanned_area?.name)
+		. += "<span class='notice'>\The [src] contains no data on [scanned_area.name].</span>"
+	else if(scanned_area)
+		. += "<span class='notice'>\The [src] contains no data on a vague area on station, you should throw it away.</span>"
+	else if(info)
+		icon_state = "slipfull"
+		. += "<span class='notice'>Wait a minute, this isn't an encrypted inspection report! You should throw it away.</span>"
```

Comment:
The icon state changes when this is examined? This would mean ghosts can affect the game.

Reply:
Removed.